cpg-neo4j/src/main/java/de/fraunhofer/aisec/cpg_vis_neo4j/Application.kt

```diff
@@ -70,14 +83,25 @@ private const val DEFAULT_SAVE_DEPTH = -1
 class Application : Callable<Int> {
     private val log: Logger
         get() = LoggerFactory.getLogger(Application::class.java)
+    // Either provide the files to evaluate or provide the path of compilation database with
+    // --json-compilation-database flag
+    @ArgGroup(exclusive = true, multiplicity = "1") lateinit var exclusive: Exclusive
+
+    class Exclusive {
+        @CommandLine.Parameters(
+            arity = "1..*",
+            description =
+                [
+                    "The paths to analyze. If module support is enabled, the paths will be looked at if they contain modules"]
+        )
+        var files: Array<String> = emptyArray()
 
-    @CommandLine.Parameters(
-        arity = "1..*",
-        description =
-            [
-                "The paths to analyze. If module support is enabled, the paths will be looked at if they contain modules"]
-    )
-    private var files: Array<String> = arrayOf(".")
+        @CommandLine.Option(
+            names = ["--json-compilation-database"],
+            description = ["Give the json compilation database file path "]
+        )
+        var jsonCompilationDatabase: String = ""
```

Comment:
Use `String? = null` instead of assigning an empty string, this is cleaner.

Reply:
Actually, you can just use `File? = null` as far as I know, pico will convert a string into a file

Reply:
Made the type as File? and added it to the latest commit.